src/onnx/parse_multinomial.cpp

```diff
@@ -0,0 +1,63 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/onnx/checks.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/make_op.hpp>
+#include <random>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_multinomial : op_parser<parse_multinomial>
+{
+    std::vector<op_desc> operators() const { return {{"Multinomial"}}; }
+
+    instruction_ref parse(const op_desc& /*opd*/,
+                          const onnx_parser& parser,
+                          const onnx_parser::node_info& info,
+                          std::vector<instruction_ref> args) const
+    {
+        int dtype = 6; // 6=in32, 7=int64
+        if(contains(info.attributes, "dtype"))
+            dtype = parser.parse_value(info.attributes.at("dtype")).at<int>();
+
+        size_t sample_size = 1;
+        if(contains(info.attributes, "sample_size"))
+            sample_size = parser.parse_value(info.attributes.at("sample_size")).at<int>();
+
+        float seed = 0.0;
```

Comment:
According to the onnx spec, we need to generate a random seed if it's not provided. That means that for your onnx test, we should always have the seed field specified in order to have deterministic results on our CI.

Reply:
Added